css/navBar.css

```diff
@@ -0,0 +1,103 @@
+nav{
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    min-height: 10vh;
+    background-color: var(--dark-grey);
+}
+
+.logo {
+    color: rgb(226, 226, 226);
```

Comment:
Use as cores "oficiais" do projeto, já padronizadas, em hexadecimal. Se for adicionar uma cor nova, coloque primeiro no arquivo `variables.css` e depois use o var() no css. Isso ajuda na manutenção =)